tests/unit_tests/test_game.py

```diff
@@ -31,6 +30,27 @@ def test_instance_logging(game_stats_service):
     logger.info.assert_called_with("{} created".format(game))
 
 
+async def test_validate_game_settings(game: Game):
+    settings = [
+        ['Victory', Victory.SANDBOX, Victory.DEMORALIZATION, ValidityState.WRONG_VICTORY_CONDITION],
+        ['FogOfWar', 'none', 'explored', ValidityState.NO_FOG_OF_WAR],
+        ['CheatsEnabled', 'true', 'false', ValidityState.CHEATS_ENABLED],
+        ['PrebuiltUnits', 'On', 'Off', ValidityState.PREBUILT_ENABLED],
+        ['NoRushOption', 20, 'Off', ValidityState.NORUSH_ENABLED],
+        ['RestrictedCategories', 1, 0, ValidityState.BAD_UNIT_RESTRICTIONS]
+    ]
+
+    for data in settings:
+        game.gameOptions[data[0]] = data[1]
```

Comment:
You should use a list of tuples rather than a list of lists, would allow you to write this instead:

```python
for data in settings:
    key, value, default, expected = data
    game.gameOptions[key] = value
    await game.validate_game_settings()
    assert game.validity is expected
...
```